sdk/keyvault/keyvault-common/src/parseWWWAuthenticate.ts

```diff
@@ -52,10 +64,10 @@ export function parseWWWAuthenticateHeader(headerValue: string): WWWAuthenticate
   const parsed = headerValue.split(pairDelimiter).reduce<WWWAuthenticate>((kvPairs, p) => {
     if (p.match(/\w="/)) {
       // 'sampleKey="sample_value"' -> [sampleKey, "sample_value"] -> { sampleKey: sample_value }
-      const [key, value] = p.split("=");
+      const [key, ...value] = p.split("=");
```

Comment:
Why is this change needed?

Reply:
The value can now contain `=` (e.g. for base64 padding), which will cause `split` to split the string into more than 2 parts. Joining the additional parts with `=` reconstructs the rest of the string after the first = sign